csaf_2.1/json_schema/csaf_json_schema.json

```diff
@@ -247,13 +247,20 @@
                 "minLength": 1
               }
             },
-            "purl": {
-              "title": "package URL representation",
-              "description": "The package URL (purl) attribute refers to a method for reliably identifying and locating software packages external to this specification.",
-              "type": "string",
-              "format": "uri",
-              "pattern": "^pkg:[A-Za-z\\.\\-\\+][A-Za-z0-9\\.\\-\\+]*\\/.+",
-              "minLength": 7
+            "purls": {
+              "title": "List of package URLs",
+              "description": "Contains a list of package URLs (purl).",
+              "type": "array",
+              "minItems": 1,
+              "uniqueItems": true,
+              "items": {
+                "title": "package URL representation",
+                "description": "The package URL (purl) attribute refers to a method for reliably identifying and locating software packages external to this specification.",
+                "type": "string",
+                "format": "uri",
+                "pattern": "^pkg:[A-Za-z\\.\\-\\+][A-Za-z0-9\\.\\-\\+]*/.+",
```

Comment:
@mprpic Was there a specific reason why you removed the `\\` in front of the `/`?

Reply:
Hmm, I think that may have been a suggested edit by my editor, but then I'm not sure why I didn't remove the double-backslashes in the character groups too. In any case, they are unnecessary in all places in this regex if the `\\-` gets moved to the end of the group, so this is correct as well:

```
                "pattern": "^pkg:[A-Za-z.+-][A-Za-z0-9.+-]*/.+",
```